lib/omnibus/builder.rb

```diff
@@ -635,6 +635,37 @@ def sync(source, destination, options = {})
     end
     expose :sync
 
+
+    def update_config_guess(target = '', version = 'master', options = {})
+      destination = File.join(software.project_dir, target)
+
+      c_g_data = {
+        :locked_version => version,
+        :locked_source => {
+          :git=>"http://git.savannah.gnu.org/r/config.git"
+        },
+        :source_type => 'git',
+        :described_version => version
+      }
+
+      config_guess_manifest = ManifestEntry.new('config_guess', c_g_data)
+
+      config_guess_dir = File.join(Config.source_dir, "config_guess-#{version}")
+      config_guess_fetcher = GitFetcher.new(config_guess_manifest, config_guess_dir, destination)
```

Comment:
So this magic scares me. I believe it would best to treat the `update_config_guess ` DSL function like we do `bundle`:
https://github.com/chef/omnibus/blob/master/lib/omnibus/builder.rb#L315-L318

Basically a project that wants to use the `bundle` DSL function should ensure Bundler is part of the project dependency tree. So in this case the assumption would be a `config` software definitoin is a dep of any software def that will call `update_config_guess ` in their build block.